edxval/api.py

```diff
@@ -576,7 +601,7 @@ def import_from_xml(xml, edx_video_id, course_id=None):
         'duration': xml.get('duration'),
         'status': 'imported',
         'encoded_videos': [],
-        'courses': [course_id] if course_id else [],
+        'courses': [{course_id: xml.get('image')}] if course_id else [],
```

Comment:
This will work. `xml.get('image')` will return `None` if `image` attribute is not present in `xml`.